File: JAVA/1768. Merge Strings Alternately.java
class Solution {
    public String mergeAlternately(String w1, String w2) {
        String s="";
        int l1=0,l2=0;
        while(l1!=w1.length() || l2!=w2.length())
        {
            if(l1==w1.length())
            s+=w2.charAt(l2++);
            else if(l2==w2.length())
            s+=w1.charAt(l1++);
            else
            {
                s+=w1.charAt(l1++);
                s+=w2.charAt(l2++);
            }
        }
        return s;
    }
}

File: C++/1768. Merge Strings Alternately.cpp
class Solution {
public:
    string mergeAlternately(string w1, string w2) {
        string a="";
        int l1=0,l2=0;
        while(l1!=w1.size() || l2!=w2.size())
        {
            if(l1==w1.size())
                a=a+w2[l2++];       
            else if(l2==w2.size())
                a=a+w1[l1++];
            else
            {
                a=a+w1[l1++];
                a=a+w2[l2++];
            }
        }
        return a;
    }
};